ReactiveCocoaTests/Swift/SignalProducerSpec.swift

```diff
@@ -72,13 +152,42 @@ class SignalProducerSpec: QuickSpec {
 		}
 
 		describe("SignalProducer.try") {
-			pending("should run the operation once per start()") {
+			it("should run the operation once per start()") {
+				var operationRunTimes = 0
+				let operation: () -> Result<String, NSError> = {
+					operationRunTimes++
+
+					return Result<String, NSError>.Success(Box("OperationValue"))
```

Comment:
or it could be just `success("OperationValue")`. LlamaKit states it's preferred because no need to deal with `Box()`.